ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml

```diff
@@ -64,6 +64,14 @@ tests:
       OCP_RELEASES: 4.12 4.16
     test:
     - ref: release-qe-tests
+- as: qe-release-gate-test-arm64
+  cron: 0 */2 * * *
+  steps:
+    env:
+      OCP_ARCH: arm64
+      OCP_RELEASES: "4.16"
```

Comment:
as I know, make update will reformat this file, i.e. remove double quote around 4.16

Reply:
Actually, it didn't, I Updated.

Reply:
Have  to change back.
```
ERROR: The following errors were found:
diff -Naupr .//ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml /tmp/tmp.vbMR2s7oDU/ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml
--- .//ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml	2024-07-25 07:17:34.542187818 +0000
+++ /tmp/tmp.vbMR2s7oDU/ci-operator/config/openshift/release-tests/openshift-release-tests-master.yaml	2024-07-25 07:17:24.622267283 +0000
@@ -6[9](https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/54749/pull-ci-openshift-release-master-ci-operator-config-metadata/1816371438177751040#1:build-log.txt%3A9),7 +69,7 @@ tests:
   steps:
     env:
       OCP_ARCH: arm64
-      OCP_RELEASES: "4.16"
+      OCP_RELEASES: 4.16
     test:
     - ref: release-qe-tests
     - ```